### ISSUE TITLE
Tracking PyTorch 1.10.0 additions / changes

### ISSUE THREAD

**Issue by NiklasGustafsson** [dotnet/TorchSharp#416]
**New functions**

torch.segment_reduce (#59951, #60018, #61141, #61266, #59521, #60379, #60379)

**Function changes**

Added channels-last support for torch.bilinear and torch.nn,MaxUnpool2d (#56322, #49984)

**Modules**

~~Added lr_scheduler.SequentialLR (#64037, #65035)~~

nn.{~~ReflectionPad3d~~, LazyInstanceNorm*d} (#59791, #60837, #61308, #60982)
nn.CrossEntropyLoss: Added support for class probability targets (#61044)
nn.CrossEntropyLoss: Added support for label smoothing (#63122)
nn.Module: Added support for arbitrary objects in state_dicts via get_extra_state() / set_extra_state() (#62976)
nn.utils.skip_init(): Added function to skip module parameter / buffer initialization (#57555)
